lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -234,7 +234,7 @@ protected function transformCollection($items, $method = 'transform') {
    */
   protected function transformCampaign($data) {
     $output = array(
-      'id' => $data->id,
+      'id' => $data->id ?: $data->nid,
```

Comment:
Had to look up this syntax cause i'd never seen it before. seems fine. somewhere in the internet i feel like it discourages shorthand syntax, because it makes the code less readable... can't seem to find it though.  Seems fine for now tho